presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragmenterUtils.java

```diff
@@ -244,6 +246,16 @@ public static Set<PlanNodeId> getTableWriterNodeIds(PlanNode plan)
                 .collect(toImmutableSet());
     }
 
+    public static Optional<Integer> getTableWriterTasks(PlanNode plan)
```

Comment:
Get the suggested number of table writer tasks from query plan, it finds the TableWriterNode, and read from its task number if scale writer field.